packages/IMAPClient-Core.package/ICEmail.class/instance/senderName.st

```diff
@@ -0,0 +1,6 @@
+accessing-header
+senderName
+
+	| fromCollection |
+	fromCollection := self from findBetweenSubStrs: '<>'.
+	^ (fromCollection at: 1 ifAbsent: [ '' ])
```

Comment:
from is a string that begins with the sender's name, followed by the email in `<>`"Der Tag - Der SPIEGEL <der-tag@newsletter.spiegel.de>". findBetweenSubstrs splits from in two entries. If the name is absent, senderName returns the email address.

Reply:
`<>` is s bit confusing, maybe we should introduce a constant (something like emailAdddressDelimeters) that returns `'<>'`

Reply:
I agree